bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -21,126 +21,31 @@ spec:
-    * `<vCenter Datastore Name>`: datastore name
-    * `<vCenter Server FQDN/IP>`: IP address or FQDN of the vCenter server
-
-    Please note that on vSphere, Windows Machine names cannot be more than 15 characters long. The MachineSet name therefore
-    cannot be more than 9 characters long, due to the way Machine names are generated from it.
-    ```
-    apiVersion: machine.openshift.io/v1beta1
-    kind: MachineSet
-    metadata:
-      labels:
-        machine.openshift.io/cluster-api-cluster: <infrastructureID>
-      name: winworker
-      namespace: openshift-machine-api
-    spec:
-      replicas: 1
-      selector:
-        matchLabels:
-          machine.openshift.io/cluster-api-cluster: <infrastructureID>
-          machine.openshift.io/cluster-api-machineset: winworker
-      template:
-        metadata:
-          labels:
-            machine.openshift.io/cluster-api-cluster: <infrastructureID>
-            machine.openshift.io/cluster-api-machine-role: worker
-            machine.openshift.io/cluster-api-machine-type: worker
-            machine.openshift.io/cluster-api-machineset: winworker
-            machine.openshift.io/os-id: Windows
-        spec:
-          metadata:
-            labels:
-              node-role.kubernetes.io/worker: ""
-          providerSpec:
-            value:
-              apiVersion: vsphereprovider.openshift.io/v1beta1
-              credentialsSecret:
-                name: vsphere-cloud-credentials
-                namespace: openshift-machine-api
-              diskGiB: 128
-              kind: VSphereMachineProviderSpec
-              memoryMiB: 16384
-              metadata:
-                creationTimestamp: null
-              network:
-                devices:
-                - networkName: "<VM Network Name>"
-              numCPUs: 4
-              numCoresPerSocket: 1
-              snapshot: ""
-              template: <Windows_VM_template
-              userDataSecret:
-                name: windows-user-data
-              workspace:
-                datacenter: <vCenter DataCenter Name>
-                datastore: <vCenter Datastore Name>
-                folder: <Path to VM Folder in vCenter> # e.g. /DC/vm/ocp45-2tdrm
-                server: <vCenter Server FQDN/IP>
-    ```
-    Example MachineSet for other cloud providers:
-    - [AWS](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md)
-    - [Azure](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-azure.md)
+    Please see the usage section of [README.md](https://github.com/openshift/windows-machine-config-operator/blob/community-4.10/README.md#usage).
 
     ### Limitations
     #### DeploymentConfigs
     Windows Nodes do not support workloads created via DeploymentConfigs. Please use a normal Deployment, or other method to
     deploy workloads.
-
-    ### Reporting issues
```

Comment:
Should we point people to our github instead?

Reply:
It's mentioned on [line 27](https://github.com/openshift/windows-machine-config-operator/pull/1161/commits/e4b4c08a71ae3092e7ea4468444a5348db8cfdb8#diff-53f141ca70f6ce0a959fa80b57ab2ac4e1e8f29689213aebce7adf958f5c71caR27)